tests/core/test_config_utils.py

```diff
@@ -117,12 +117,7 @@ class DummyDataClass:
 
     @pytest.mark.unit
     def test_ptl_config(self):
-        PTL_DEPRECATED = [
-            'gpus',
-            'num_processes',
-        ]
-
-        result = config_utils.assert_dataclass_signature_match(ptl.Trainer, TrainerConfig, ignore_args=PTL_DEPRECATED)
+        result = config_utils.assert_dataclass_signature_match(ptl.Trainer, TrainerConfig)
```

Comment:
Hmm lets keep empty list of PTL_DEPRECATED = [] here and pass the arg. We may have to use it in the futuer.

Reply:
Fixed.